### PR TITLE
tags that comply with the expected format

### PR DESCRIPTION
**Pull request by replay**
the expected format for tags is `k=v`

### CODE REVIEW

File: fakemetrics.go [hunk 1]
```diff
@@ -196,15 +196,17 @@ func runMultiplied(orgs, keysPerOrg, metricPeriod, flushPeriod, offset, speedup
...
-					Metric:   "some.id.of.a.metric",
+					Name:     name,
+					Metric:   metric,
 					OrgId:    o,
 					Interval: metricPeriod,
 					Value:    0,
 					Unit:     "ms",
 					Mtype:    "gauge",
-					Tags:     []string{"some_tag", "ok", fmt.Sprintf("k:%d", k)},
+					Tags:     []string{"some=tag", fmt.Sprintf("name=%s", name), fmt.Sprintf("metric=%s", metric)},
```
Comment (Dieterbe) [inline fakemetrics.go:209]: why tag every metric with `metric=some.id.of.a.metric` ? that doesn't seem very realistic. maybe tag it `fmt.Sprintf("id=%d",k)` instead

Comment (Dieterbe) [reply fakemetrics.go:209]: ditto for name tag

Comment (replay) [reply fakemetrics.go:209]: Regarding the `name` tag, that's what carbon does, so that's just consistent. Then regarding the `metric` tag, i basically just wanted to have some tags which refer to a small number of series and some that refer to a large number of series, because that makes testing different edge cases easier.
But even more important is that the format is valid so they can actually be used as test data
